src/checkpoint.h

```diff
@@ -17,6 +17,6 @@ You should have received a copy of the GNU General Public License
 along with mfaktc (mfakto).  If not, see <http://www.gnu.org/licenses/>.
 */
 
-void checkpoint_write(unsigned int exp, int bit_min, int bit_max, unsigned int cur_class, int num_factors);
-int checkpoint_read(unsigned int exp, int bit_min, int bit_max, unsigned int *cur_class, int *num_factors, int verbosity);
+void checkpoint_write(unsigned int exp, int bit_min, int bit_max, unsigned int cur_class, int num_factors, char* factors_string, unsigned long long int bit_level_time);
+int checkpoint_read(unsigned int exp, int bit_min, int bit_max, unsigned int *cur_class, int *num_factors, char* factors_string, unsigned long long int bit_level_time, int verbosity);
```

Comment:
There is a discrepancy between the header and the implementation in the type of `bit_level_time`. The header wants a scalar, the implementation wants a pointer. Please use a strict modern compiler and fix the warnings.

Reply:
This was likely a typo on my part, my apologies. MSC didn't seem to give a warning on build in visual studio, not sure if this is due to the project settings or what. But it did have a barely noticeable green ribbon on the actual code here. Will be fixed in the next commit.